### DISCUSSION TITLE
Exclude Wallets From Connected List

### DISCUSSION THREAD

**Discussion by petermazzocco** [family/connectkit#348] [Q&A]
Is there a way to exclude a wallet from the list of available wallets to connect to? I do not want to allow the user to connect to Coinbase Wallet so it would be nice to exclude certain wallets!

**Comment by lochie** [comment]
This feature is in public beta ✨
https://docs.family.co/connectkit/wallets

If you only wish to exclude a certain wallet you can always filter it out of the connectors.

```ts
const ckConfig = getDefaultConfig({
  // your config
});

const config = createConfig({
  ...ckConfig,
  connectors: ckConfig.connectors.filter((c) => c.id !== 'coinbaseWallet')
});
```
